packages/react/src/hooks/use-last-factor.ts

```diff
@@ -0,0 +1,79 @@
+import { useEffect, useMemo, useCallback } from "react";
+import { isBrowser } from "@slashid/react-primitives";
+import { Handle, isFactor } from "../domain/types";
+import { useConfiguration } from "./use-configuration";
+import { useSlashID } from "./use-slash-id";
+import { Factor } from "@slashid/slashid";
+
+export const STORAGE_LAST_FACTOR_KEY = (oid: string) =>
+  `@slashid/LAST_FACTOR/${oid}`;
+
+type UseLastFactor = () => {
+  lastFactor: Factor | undefined;
+};
+
+type SuccessEvent = {
+  authenticationFactor: Handle | undefined;
+};
+
+export const useLastFactor: UseLastFactor = () => {
```

Comment:
Can we annotate the return value of this hook?

Reply:
It is annotated by `UseLastHandle` type